src/local_scheduler/local_scheduler_algorithm.cc

```diff
@@ -540,6 +537,15 @@ void fetch_missing_dependency(LocalSchedulerState *state,
      * the object becomes available locally. It will get freed if the object is
      * subsequently removed locally. */
     ObjectEntry entry;
+    /* If the task is for an actor, and the missing object is a dummy object,
+     * then we must generate it locally by executing the corresponding task.
+     * All other objects may be requested from another plasma manager. */
+    if (TaskSpec_is_actor_task(task_entry_it->spec) &&
```

Comment:
Is this if statement ever true? It seems to me like we're still never calling `fetch_missing_dependencies` on actor tasks. Or are we somewhere?

Reply:
Oh my mistake, I see that it happens through `queue_task_locally`.